doc/source/text.rst

```diff
@@ -168,28 +168,34 @@ Extracting Substrings
 
 .. _text.extract:
 
-The method ``extract`` (introduced in version 0.13) accepts `regular expressions
-<https://docs.python.org/2/library/re.html>`__ with match groups. Extracting a
-regular expression with one group returns a Series of strings.
+.. versionadded:: 0.13.0
 
-.. ipython:: python
+.. warning::
+
+   In version 0.18.0, ``extract`` gained the ``expand`` argument. When
+   ``expand=False`` (default) it returns a ``Series``, ``Index``, or
+   ``DataFrame``, depending on the subject and regular expression
```

Comment:
I though the plan was to make ``expand=None`` the default. Then you can provide a deprecation warning, and support ``expand=False`` (return Series,Index,or DataFrame as appropriate to the data), and have the default actually be ``expand=True`` going forward. This provides back-compat with a warning, and explicit behavior.